sqlcon/error.go

```diff
@@ -43,6 +47,8 @@ func HandleError(err error) error {
 		switch err.Code {
 		case "23505": // "unique_violation"
 			return errors.Wrap(ErrUniqueViolation, err.Error())
+		case "40001": // "serialization_failure"
```

Comment:
Do we only care about PostgreSQL for now?

Reply:
See https://github.com/ory/x/pull/129#issuecomment-603427081